src/tools/perf/api/libperf.h

```diff
@@ -236,6 +236,24 @@ typedef struct ucx_perf_rte {
 } ucx_perf_rte_t;
 
 
+typedef struct {
+    /* Whether DPU offloading daemon is configured */
+    int                     is_daemon_mode;
+    /* Whether to keep daemon alive at the end of the test run */
+    int                     is_keep_alive;
```

Comment:
maybe rename to `is_keep_running` to avoid confusion with keepalive protocols?

Reply:
ok, renamed, + all the related variables and comments